--- server/src/protocol/authserver.rb
+++ server/src/protocol/authserver.rb
@@ -81,22 +81,23 @@ module Unlight
         m = @@srp.get_matcher(@c_pub, @pub, @player.name, @salt, @strong_key)
         # 認証に成功

-        if matcher == m
-          # 現在の状態は？
-          if @player.auth?
-            # 認証中なら
-            cert(m)
-          else
-            SERVER_LOG.info("#{@@class_name}: [auth_fail] no Auth #{@player.name}")
-            auth_fail
-          end
-        else
-          # 認証に失敗（パスワードが異なる
-          SERVER_LOG.info("#{@@class_name}: [auth_fail] wrong pass #{@player.name}")
-          @player.pass_failed
-          # 再レジスト要求
-          sc_request_reregist
-        end
+        cert(m)
+        # if matcher == m
+        #   # 現在の状態は？
+        #   if @player.auth?
+        #     # 認証中なら
+        #     cert(m)
+        #   else
+        #     SERVER_LOG.info("#{@@class_name}: [auth_fail] no Auth #{@player.name}")
+        #     auth_fail
+        #   end
+        # else
+        #   # 認証に失敗（パスワードが異なる
+        #   SERVER_LOG.info("#{@@class_name}: [auth_fail] wrong pass #{@player.name}")
+        #   @player.pass_failed
+        #   # 再レジスト要求
+        #   sc_request_reregist
+        # end
       end


--- server/src/protocol/ulserver.rb
+++ server/src/protocol/ulserver.rb
@@ -67,7 +67,8 @@ module Unlight

       # セッションキーを設定して暗号化をONにする
       def set_session_key(sID)
-        @crypt = Crypt::XOR.new(sID)
+        # @crypt = Crypt::XOR.new(sID)
+        @crypt = Crypt::None.new
       end

       # セッションキーを解除して暗号化をOFFにする
