--- client/src/view/scene/common/StoryTextField.as
+++ client/src/view/scene/common/StoryTextField.as
@@ -98,6 +98,7 @@ package view.scene.common
         private var _initFunc:Function;

         private var _titleHtml:String;
+        private var _scroller:DialogScroll;


         /**
@@ -173,6 +174,8 @@ package view.scene.common
             _maskBitmap = new Bitmap(_maskBitmapData);
             _textAreaBitmap = new Bitmap(_textAreaBitmapData);
             _maskBitmap.scrollRect = new Rectangle(0,0,width,_visibleHeight)
+            _scroller = new DialogScroll(_maskBitmap);
+
             log.writeLog(log.LV_FATAL, this, "setBitmap end");
         }

@@ -200,7 +203,7 @@ package view.scene.common
              // 移動量
              var lineNum:int = Math.abs(_infoLines[line][0] - _maskBitmap.scrollRect.y);
              log.writeLog(log.LV_FATAL, this, "Scrolling to ",line,lineNum,_infoLines[line][0]);
-             return new TweenerThread(_maskBitmap, {_scrollRect_y: _infoLines[line][0], transition:"easeOutCirc", time:0.0005*lineNum+0.0002})
+             return new TweenerThread(_scroller, {scrollTop: _infoLines[line][0], transition:"easeOutCirc", time:0.0005*lineNum+0.0002})
          }

         // 自動スクロールするスレッド
@@ -238,6 +241,26 @@ package view.scene.common
     }
 }

+class DialogScroll {
+  private var _scrollTop:Number = 0.0;
+  private var _mask:Bitmap;
+
+  public function DialogScroll(mask:Bitmap) {
+    _mask = mask;
+  }
+
+  public function get scrollTop():Number {
+    return _scrollTop;
+  }
+
+  public function set scrollTop(value:Number):void {
+    _scrollTop = value;
+    var rect:Rectangle = _mask.scrollRect;
+    rect.y = _scrollTop
+    _mask.scrollRect = rect
+  }
+}
+
 import flash.display.*;
 import flash.geom.*;
 import flash.text.*;
